pkg/performanceprofile/controller/performanceprofile/components/kubeletconfig/kubeletconfig_test.go

```diff
@@ -169,10 +170,15 @@ var _ = Describe("Kubelet Config", func() {
 
 			manifest := string(y)
 
-			memoryAvaialable := "memory.available: " + eviction.DefaultEvictionHard[evictionHardMemoryAvailable]
-			nodefsAvailable := "nodefs.available: " + eviction.DefaultEvictionHard[evictionHardNodefsAvaialble]
-			imagefsAvailable := "imagefs.available: " + eviction.DefaultEvictionHard[evictionHardImagefsAvailable]
-			nodefsInodesFree := "nodefs.inodesFree: " + eviction.DefaultEvictionHard[evictionHardNodefsInodesFree]
+			// When this test fails, compare whether the defaults in kubernetes documentation
```

Comment:
I would add a sentence about the existance of the `update-k8s-eviction-defaults.sh` script that might help when this test fails.

Reply:
good idea, adding